public/app/features/explore/TraceView/TraceView.test.tsx

```diff
@@ -20,6 +20,13 @@ describe('TraceView', () => {
     expect(header).toHaveLength(1);
   });
 
+  it('does not render anything on missing trace', () => {
+    // Simulating Explore's access to empty response data
+    const trace = [][0];
+    const wrapper = shallow(<TraceView trace={trace} splitOpenFn={() => {}} />);
+    expect(wrapper.isEmptyRender()).toBeTruthy();
+  });
+
```

Comment:
small nit: we're trying to move away from enzyme in favor of @testing-library/react. Is there some way we could take a first small step towards using @testing-library/react instead of adding to our enzyme tech debt?

Reply:
Tried my best to mimic `isEmptyRender()`